test/Autofac.Extras.FakeItEasy.Test/AutoFakeFixture.cs

```diff
@@ -193,6 +193,19 @@ public void ConfigureFakeOverridesStrict()
             }
         }
 
+        [Fact]
+        public void ReturnsInstanceFromChildScope()
+        {
+            using (var fake = new AutoFake())
+            {
+                var bar = fake.Resolve<IBar>();
+                fake.Provide(new Baz());
+                var bar2 = fake.Resolve<IBar>();
+
+                Assert.Equal(bar, bar2);
```

Comment:
Possibly better to assert that the instances are the same, rather than just equal?